core/indigo-core/molecule/src/cml_saver.cpp

```diff
@@ -361,6 +361,10 @@ void CmlSaver::_addMoleculeElement(XMLElement* elem, BaseMolecule& mol, bool que
                 }
                 else
                 {
+                    if (pos.x == 0)
+                        pos.x = 0;
+                    if (pos.y == 0)
+                        pos.y = 0;
```

Comment:
If a value already is zero, set to zero.
Strange logic block that does nothing. 
Looking to above clause, it can be removed.

Reply:
It was desperate attemp to eliminate negative zeros from some coordinates in xml, which have started to appear after updating TinyXML with TinyXML2. It is an example of xml, produced by TinyXML2:

<img width="793" alt="image" src="https://user-images.githubusercontent.com/78915761/161036573-6025fa3e-0a48-4ed5-a7e7-8d97ca879b07.png">

Reply:
C++ float numbers really have some peculiarities regarding `-0f`. 
Thinking about coordinates, they can hardly be essential, can't they?
Anyway, if the problem comes from reading XML, the correction should be made in the corresponding layer.
`MoleculeCdxmlLoader` ?

Reply:
So I'd propose the code like this:
```
    if (pos.x == -0.f)
        pos= 0.f;
```
Or even better (quasi quantization around zero):
```
    if (fabs(pos.x) < FLT_MIN)
        pos= 0.f;
```

Reply:
Thank you, @SunFellow! And extend it to the rest coordinates just in case, right?

Reply:
Yes, of course.